src/main/kotlin/snyk/common/lsp/LanguageServerWrapper.kt

```diff
@@ -405,6 +406,7 @@ class LanguageServerWrapper(
             integrationVersion = pluginInfo.integrationVersion,
             authenticationMethod = authMethod,
             enableSnykOSSQuickFixCodeActions = "true",
+            enableDeltaFindings = ps.isDeltaFindingsEnabled(),
```

Comment:
This line is unnecessary, as the default is already set when instantiating the settings class. As the default is exactly the same, this line is redundant.

Reply:
Fixed!